code/modules/organs/internal/_internal.dm

```diff
@@ -48,9 +47,56 @@
 /obj/item/organ/internal/is_usable()
 	return ..() && !is_broken()
 
-/obj/item/organ/internal/heal_damage(amount , var/natural = TRUE)
+/obj/item/organ/internal/heal_damage(amount, natural = TRUE)
 	if (natural && !can_recover())
 		return
 	if(owner.chem_effects[CE_BLOODCLOT])
 		amount *= 1 + owner.chem_effects[CE_BLOODCLOT]
-	damage = between(0, damage - round(amount, 0.1), max_damage)
\ No newline at end of file
+	damage = between(0, damage - round(amount, 0.1), max_damage)
+
+
+// Gets the limb this organ is located in, if any
+/obj/item/organ/internal/proc/get_limb()
```

Comment:
AFAIR, that organ is already hold in `parent` var.

Reply:
That's only true for `external` organs, and I'm not about to start tampering with that code. At least not yet.